scss/components/_transitions.scss

```
@@ -0,0 +1,62 @@
+.transition-area {
+  background-color: #ccc;
+  position: relative;
+
+  &::after {
+    content: '';
+    display: block;
+    padding-top: 20%;
+  }
+
+  &:hover {
+    .transition-child {
+      left: calc(100% - 50px);
```

Please apply this suggestion
```suggestion
      left: calc(100% - 60px);
```
your transition child has grown to 60 in width + height, then this transition should also reflect that